folioclient/FolioClient.py

```diff
@@ -27,13 +29,18 @@ def __init__(self, okapi_url, tenant_id, username, password, ssl_verify=True):
         self.username = username
         self.password = password
         self.ssl_verify = ssl_verify
-        self.login()
-        self.okapi_headers = {
-            "x-okapi-token": self.okapi_token,
+        self.httpx_client = None
+        self.refresh_token = None
+        self.cookies = None
+        self.okapi_token_expires = None
+        self.okapi_token_duration = None
+        self.refresh_token_expires = None
+        self.refresh_token_duration = None
```

Comment:
It looks like these are never used, only updated. Are there plans to use them? Perhaps worth a comment in the code if they are a placeholder for future work.

Reply:
Yeah, I realized that earlier today. May pull them out.

Reply:
I keep getting distracted with thoughts about the TTL of the Refresh Token, and forgetting that it gets replaced whenever the Access Token is updated. This makes me think two things:
1. I might be getting confused by storing the Access Token in `okapi_token`, which is maybe a confusion based on the language in the proposal; and
2. maybe the README or a class docstring should point to some official RTR documentation as an aid to whoever is trying to understand the login/authentication process(es) that this class is encapsulating.

Reply:
I'm going to pull out the refresh handling altogether. Given we have the username and password available, just logging in again should suffice, and that's what the developers that implemented RTR recommend for scripted clients.

Reply:
I'm wondering about some future scenario where we don't want to keep the credentials around in memory, but would want purge them. Maybe long-running processes in Python, say an edge module or a local application. 
If that seems a likely future scenario, maybe leave the refresh handling available in a branch so in can be re-added should the need arise.

Reply:
I agree. I won't delete this branch, and that implementation is in the commit history if we need it.